debezium-connector-mysql/src/test/java/io/debezium/relational/history/KafkaDatabaseHistoryTest.java

```diff
@@ -378,6 +382,23 @@ public void shouldValidateMandatoryValues() {
                 "database.history.kafka.bootstrap.servers",
                 "database.history.kafka.recovery.poll.interval.ms",
                 "database.history.connector.id",
-                "database.history.kafka.recovery.attempts"));
+                "database.history.kafka.recovery.attempts",
+                "database.history.kafka.query.timeout.ms"));
+    }
+
+    @Test
```

Comment:
Could you add the `@FixFor` annotation with the right issue key?

Reply:
We generally use it to tag the corresponding issue. A better name would be `@IssueKey`, but here we are :)

Reply:
Used `@FixFor`.